src/app/component/auth/components/restore-password/restore-password.component.html

```diff
@@ -37,6 +37,7 @@ <h2>{{'user.auth.forgot-password.fill-form' | translate}}</h2>
     </button>
     <span class="or-use-google">{{'user.auth.forgot-password.or-signin' | translate}}</span>
     <app-google-btn (click)="signInWithGoogle()"
+                    class="google-sing-in_button"
```

Comment:
In my view such class name with snake case looks unusual.

Reply:
@luchward I see myself in this sin too, but as an excuse I can give the example of the bem methodology (http://getbem.com/naming/). Maybe it's not all of it but I think we can think together about the common name of the classes and then write it in our guide?